src/components/search/curated/default-cta.tsx

```diff
@@ -7,7 +7,7 @@ const DefaultCTA: React.FC<{location: string}> = ({location}) => {
     <ExternalTrackedLink
       eventName="clicked epic react banner"
       params={{location}}
-      className="block md:col-span-4 w-full h-full overflow-hidden border-0 border-gray-100 relative text-center"
+      className="block md:col-span-4 lg:w-full h-full overflow-hidden border-0 border-gray-100 relative text-center"
```

Comment:
Add `lg` breakpoint here to avoid empty area clickability:
![Image 2022-04-21 at 12 08 54 PM](https://user-images.githubusercontent.com/1519448/164421409-c9435536-6cae-4f5b-951e-7a8d68843020.jpg)